ext/scheduler/airflow2/airflow.go

```diff
@@ -233,7 +230,7 @@ func (s *scheduler) ListJobs(ctx context.Context, namespace models.NamespaceSpec
 		return jobs, nil
```

Comment:
`opts.OnlyName` is always true in the codebase, should we remove this check? and the following code